source-hubspot-native/tests/test_snapshots.py

```diff
@@ -20,19 +20,6 @@ def test_capture(request, snapshot):
     assert result.returncode == 0
     lines = [json.loads(l) for l in result.stdout.splitlines()[:50]]
 
-    for l in lines:
-        _collection, record = l[0], l[1]
-
-        for m in ["properties", "propertiesWithHistory"]:
-            for prop in [
-                "hs_time_in_lead",
```

Comment:
I'm curious as to why we don't need to do this anymore?

Reply:
Im not sure what this did, but it was made to remove these specific timestamps specifically from v3 objects properties, i didnt deem it necessary as there were no comments explaining why ( my idea was that this was probably breaking tests? )

Reply:
OK yeah, I guess if the tests work repeatedly without it then it shouldn't be a problem.